src/metabase/upload.clj

```diff
@@ -678,7 +690,9 @@
 ;;; +--------------------------------------------------
 
 (mu/defn append-csv!
-  "Main entry point for appending to uploaded tables with a CSV file."
+  "Main entry point for appending to uploaded tables with a CSV file.
+  This will create an auto-incrementing primary key (auto-pk) column in the table for drivers that supported uploads
+  before auto-pk columns were introduced by metabase#36249."
```

Comment:
Why not put a clickable link for the issue / PR?

Reply:
A clickable link is longer so we generally just link to issues/PRs like this. I think those on emacs have set up an automation to open these as links quickly.